src/EFCore.PG.NTS/Storage/Internal/NpgsqlNetTopologySuiteTypeMappingSourcePlugin.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using GeoAPI.Geometries;
+using JetBrains.Annotations;
+using Microsoft.EntityFrameworkCore.Storage;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Infrastructure.Internal;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Internal;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Utilities;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal
+{
+    public class NpgsqlNetTopologySuiteTypeMappingSourcePlugin : IRelationalTypeMappingSourcePlugin
+    {
+        // Note: we reference the options rather than copying IsGeographyDefault out, because that field is initialized
+        // rather late by SingletonOptionsInitializer
+        readonly INpgsqlNetTopologySuiteOptions _options;
+
+        readonly HashSet<string> _spatialStoreTypes = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
```

Comment:
Is this ever expanded? If not (or if growth is unlikely), is there a perf argument to prefer array iteration instead of the `HashSet`?

Reply:
No perf aspect here, but I changed the code to simply compare directly against `geometry` and `geography` for simplicity.